airflow/models/baseoperator.py

```diff
@@ -1081,21 +1082,22 @@ def get_extra_links(self, dttm, link_name):
     @classmethod
     def get_serialized_fields(cls):
         """Stringified DAGs and operators contain exactly these fields."""
-        if not cls._serialized_fields:
-            cls._serialized_fields = frozenset(
+        if not cls.__serialized_fields:
+            cls.__serialized_fields = frozenset(
                 vars(BaseOperator(task_id='test')).keys() - {
                     'inlets', 'outlets', '_upstream_task_ids', 'default_args', 'dag', '_dag'
-                } | {'_task_type', 'subdag', 'ui_color', 'ui_fgcolor', 'template_fields'}
-            )
-        return cls._serialized_fields
+                } | {'_task_type', 'subdag', 'ui_color', 'ui_fgcolor', 'template_fields'})
+
+        return cls.__serialized_fields
 
 
+@attr.s(auto_attribs=True)
 class BaseOperatorLink(metaclass=ABCMeta):
     """
     Abstract base class that defines how we get an operator link.
     """
 
-    operators: List[Type[BaseOperator]] = []
+    operators = []   # type: List[Type[BaseOperator]]
```

Comment:
Don't think you wanted to "revert" this change when resolving conflicts.

Reply:
Fixed